modules/ROOT/pages/upgrade-self-managed.adoc

```diff
@@ -0,0 +1,87 @@
+= Upgrade Runtime Fabric for Self-Managed Kubernetes
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, you need to upgrade Runtime Fabric components and the `rtfctl` command-line utility.
+
+== About Component Upgrades
+
+Runtime Fabric component upgrades involve the internal services that run within the appliance used to facilitate communication to the Anypoint control plane, provide load balancing, and log and metrics forwarding.
+
+Runtime Fabric component upgrades are applied within Runtime Manager via a button that's visible when an upgrade is made available.
+
+Runtime Fabric component upgrades do not require adding a node.
+
+== About `rtfctl` Command-Line Utility Upgrades
+The `rtfctl` command-line utility is used to manage Runtime Fabric and perform appliance and node configuration upgrades.
+
+You upgrade the `rtfctl` utility from the command line.
+
+It is important to use the most current version of the `rtfctl` command-line utility.
+
+== Determine the Component Versions You Are Using
+
+From your local workstation (configured with a kubeconfig file) or a support bastion, run the following command:
+```
+.rtfctl version
+```
+
+Output includes:
+
+* The `rtfctl` command-line utility version in the `rtfctl` field
+* The Runtime Fabric component version in the `agent` field
+
+For example:
+```
+$ ./rtfctl version
+COMPONENT    VERSION
+rtfctl       0.3.122
+agent        1.9.6
+kubernetes   1.19.6-eks-49a6c0
+
+```
+
+[NOTE]
+The Runtime Fabric component version that is in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
+
+== Apply the Upgrades
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, you:
+
+. Upgrade Runtime Fabric Components
+. Upgrade the `rtfctl` command-line utility
+
+Before upgrading your production environment, perform these steps on a nonproduction Runtime Fabric to verify compatibility with your environment. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.
```

Comment:
```suggestion
Before upgrading your production environment, perform these steps on a nonproduction Runtime Fabric instance to verify compatibility with your environment. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.
```
Currently, this working in sandbox, testing, and then moving to production thing is pretty hidden, and it's pretty important. I would mention it earlier, as of equal importance as the two upgrades.

Reply:
Also, this text is a little confusing, because it sounds like you perform the upgrade in the sandbox, test, and then later _move_ the upgrade (or _repeat_ "these steps") in production during downtime.